README.md

```diff
@@ -22,7 +22,7 @@ yarn add @mux/mux-node
 ```
 
 ## Releases
-The latest **stable** release is `1.1.1`
+The latest **stable** release is `2.0.0`
```

Comment:
Ooh if this is a major version bump... let's just kill the lower-cased versions of things?